.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: release
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-go@v3
+        with:
+          go-version-file: 'go.mod'
+      - name: Set GOLANG_VERSION
```

Comment:
not sure that this is needed as I don't see the GOLANG_VERSION being used.. is it merely for console output?